Libraries/react-native/react-native.js

```diff
@@ -20,6 +20,7 @@
 var ReactNative = Object.assign(Object.create(require('React')), {
   // Components
   ActivityIndicatorIOS: require('ActivityIndicatorIOS'),
+  ART: require('ReactNativeART'),
```

Comment:
Yeah that's great. Would be good to get to a state where you pay for only what you use, for most of the components & modules.